packages/elements/src/components/ino-dialog/ino-dialog.tsx

```diff
@@ -52,6 +53,34 @@ export class Dialog implements ComponentInterface {
    */
   @Prop() open = false;
 
+  /**
+   * Adds a headline to the `ino-dialog`
+   */
+  @Prop() headline?:string;
+
+  /**
+   * Adds a text to the body of the `ino-dialog`
+   */
+  @Prop() description?:string;
+
+  /**
+   * Adds a button with the given text to close the `ino-dialog`
+   */
+  @Prop() cancelbutton?:string;
+
+  /**
+   * Adds a button with the given text to proceed with an action`
+   */
+  @Prop() actionbutton?:string;
```

Comment:
I would suffix these props with `Text` (e.g. `headerText`) to imply that its only the string can be provided